src/go/k8s/controllers/redpanda/console_controller.go

```diff
@@ -326,7 +325,7 @@ func (r *ConsoleReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&corev1.Service{}).
 		Owns(&netv1.Ingress{}).
 		Watches(
-			&source.Kind{Type: &vectorizedv1alpha1.Cluster{}},
```

Comment:
I'm not sure if this change is working as unit tests are failing

Reply:
Perhaps not, but controller-runtime changed this function and this looks like it should be right: https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.16.2/pkg/builder#Builder.Watches

Do you see a mistake?

Reply:
Your change seems to be aligned with controller runtime.